modules/about-bare-metal-hosts-and-nodes.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/managing-bare-metal-hosts.adoc
+
+[id="about-bare-metal-hosts-and-nodes_{context}"]
+= About bare metal hosts and nodes
+
+In order to provision a {op-system-first} bare metal host as a node in your cluster, you must first create a `MachineSet` CR object that corresponds to the bare metal host hardware. Bare metal host machine sets describe infrastructure components specific to your configuration. You apply specific Kubernetes labels to these machine sets and then update the infrastructure components to run on only those machines.
```

Comment:
s/In order to/To/
s/you must first/first/
s/CR/configuration resource (CR)/
s/You apply/Apply/

Reply:
re: "s/CR/configuration resource (CR)/", is this a typo? do you mean custom resource (CR)?